smart-contracts/contracts/range-middleware/src/state.rs

```diff
@@ -12,3 +13,7 @@ pub struct NewRange {
 pub const PENDING_RANGES: Map<Addr, NewRange> = Map::new("pending_ranges");
 pub const RANGE_SUBMITTER_ADMIN: Item<Addr> = Item::new("range_submitter_admin");
```

Comment:
Either we should keep the admin fields and migrate them, or Initialize new ones and clean these up so that our state stays clean

Reply:
Since the migrate code already exists for this, we probably want to inline this or add a #[deprecate] so we properly deprecate these fields

Reply:
imo, we should remove them completely, i.e.
- Remove the old items here
- in migrate read the values from locally defined `Item::new("...")` and clean the associated storage.

Reply:
resolved [69b1120](https://github.com/quasar-finance/quasar/pull/649/commits/69b1120dee0603a9f8f86a3c4c8208786b52dd8f)